projects/hee-shared/src/lib/hotjar/hotjar.module.ts

```diff
@@ -0,0 +1,52 @@
+import { NgModule, InjectionToken, Inject, Optional, SkipSelf, ModuleWithProviders } from '@angular/core';
+import { InjectScript } from '../utilities.functions';
+import { DOCUMENT } from '@angular/common';
+
+export interface HotjarConfig {
+  hotjarid: number;
```

Comment:
is it possible to camelcase this interface?

Reply:
Revised all and camel cased across using angular style guide: https://angular.io/guide/styleguide